devito/symbolics/manipulation.py

```diff
@@ -286,12 +286,12 @@ def pow_to_mul(expr):
         except TypeError:
             # E.g., a Symbol, or possibly a generic expression
             return expr
-        if exp > 10 or exp < -10 or int(exp) != exp or exp == 0:
+        if exp > 10 or exp < -10 or exp == 0:
             # Large and non-integer powers remain untouched
```

Comment:
"non-integer" part of this comment needs moving to the `elif`

Reply:
good catch, fixing